frontend/public/components/machine-config-pool.tsx

```diff
@@ -246,12 +246,12 @@ export const MachineConfigPoolDetailsPage: React.SFC<any> = (props) => (
 );
 
 const tableColumnClasses = [
-  classNames('col-lg-3', 'col-xs-6', 'col-sm-4'),
-  classNames('col-lg-5', 'hidden-md', 'hidden-sm', 'hidden-xs'),
-  classNames('col-lg-1', 'col-sm-2', 'col-xs-3', 'pf-u-w-10-on-lg'),
-  classNames('col-lg-1', 'col-sm-2', 'hidden-xs', 'pf-u-w-10-on-lg'),
-  classNames('col-lg-1', 'hidden-sm', 'hidden-xs', 'pf-u-w-10-on-lg'),
-  classNames('col-lg-1', 'col-sm-2', 'col-xs-3', 'pf-u-w-10-on-lg'),
+  classNames('pf-u-w-25-on-lg pf-u-w-33-on-md pf-u-w-50-on-sm'),
+  classNames('pf-m-hidden pf-m-visible-on-xl pf-u-w-40-on-xl'),
```

Comment:
```suggestion
  classNames('pf-m-hidden pf-m-visible-on-xl pf-u-w-50-on-xl'),
```
with `pf-u-w-40-on-xl`

![image](https://user-images.githubusercontent.com/9964343/122553610-88bfc980-d055-11eb-826c-55c7502bab49.png)

with `pf-u-w-50-on-xl`:
![image](https://user-images.githubusercontent.com/9964343/122553701-a55c0180-d055-11eb-8383-c3a89d43f57b.png)

Reply:
You could also remove `classNames` from this file.

Reply:
updated